### ISSUE TITLE
[eslint-plugin-lit-a11y] aria-attr-valid-value fails on this.foo

### ISSUE THREAD

**Issue by thepassle** [open-wc/open-wc#1889]
This passes:
```js
    { code: 'html`<div aria-valuemax=${foo} />`' },
```

This fails, but should pass:
```js
    { code: 'html`<input aria-valuemax=${this.max} />`' },
```

I think something is not going right with the placeholder somehow, because the expression `${this.max}` should be substituted with a placeholder starting with `{{`, and be caught here:

https://github.com/open-wc/open-wc/blob/7ac9b273f6b97ac2cdf8f12972c08a38ec767808/packages/eslint-plugin-lit-a11y/lib/rules/aria-attr-valid-value.js#L125

I _think_ it might be a missing case that we dont cover here, but im not sure:
https://github.com/open-wc/open-wc/blob/7ac9b273f6b97ac2cdf8f12972c08a38ec767808/packages/eslint-plugin-lit-a11y/template-analyzer/util.js#L219

ASTExplorer is your friend 🙂 
https://astexplorer.net/#/gist/e7e7299d93a75e395402cdb42d40b97c/2e6ec797100db941ad68a04c59a342b676b7b150

**EDIT:** The same seems to be the case for ternaries in expressions, e.g.: `<input aria-disabled=${this.disabled ? 'true' : 'false'} />` I'm assuming the same problem is the case there.
